package.json

```diff
@@ -55,7 +55,11 @@
         "onView:aws.cdk.explorer",
         "onCommand:aws.refreshCdkExplorer",
         "onCommand:aws.aboutToolkit",
-        "onLanguage:asl"
+        "onLanguage:asl",
+        "onLanguage:javascript",
+        "onLanguage:python",
+        "onLanguage:csharp",
+        "onLanguage:yaml"
```

Comment:
Technically you could limit this to `*template.yaml|*template.yml`. I had that in the original change.

Reply:
Does that work for `onLanguage`? Thought that was just for `workspaceContains` which would work but would immediately activate the extension if it saw a `template.yaml` anywhere in the workspace. https://code.visualstudio.com/api/references/activation-events#workspaceContains

Reply:
Good point. Although presumably a user with a `*template.yml` in their workspace would want it activated more often than a user looking at possibly unrelated yaml files.

Reply:
I think I'll stick with what I have for now but if others on the PR think the same way (template.yaml/template.yml in the workspace deserves an instant activation) then I can go ahead and pivot.